packages/lib-user/src/components/shared/Layout/Layout.js

```diff
@@ -0,0 +1,151 @@
+'use client'
+
+import { Box } from 'grommet'
+import { node } from 'prop-types'
+import styled, { css } from 'styled-components'
+
+const PageContainer = styled(Box)`
+  display: grid;
+  grid-template-columns: 0 0 1fr 0 0;
+  grid-template-rows: 0 0 1fr;
+  min-height: 90vh;
+
+  @media (48rem <= width < 80rem) {
+    grid-template-columns: 0 2rem minmax(44rem, 76rem) 2rem 0;
+    grid-template-rows: 50px 50px 1fr;
+  }
+
+  @media (80rem <= width) {
+    grid-template-columns: auto minmax(2rem, 5rem) minmax(74rem, 80rem) minmax(2rem, 5rem) auto;
+    grid-template-rows: 70px 70px 1fr;
+  }
+`
+
+const PageHeader = styled(Box)`
+  grid-column: 1 / 6;
+  grid-row: 1 / 3;
+`
+
+const PageLeftColumn = styled(Box)`
+  position: relative;
+  grid-column: 2 / 3;
+  grid-row: 3 / 4;
+
+  &::after {
+    content: '';
+    position: absolute;
+    top: 0;
+    right: -1px;
```

Comment:
```suggestion
    right: 0;
```

This `1px` difference is noticeable when zooming in:
<img width="94" alt="Screenshot 2024-01-16 at 11 31 12 AM" src="https://github.com/zooniverse/front-end-monorepo/assets/23665803/dbb697ca-2f68-4b80-b7ec-581a97f4329c">